source/Projectile.cpp

```diff
@@ -284,6 +272,25 @@ shared_ptr<Ship> Projectile::TargetPtr() const
 }
 
 
+//contains statement needed in multiple constructors.
```

Comment:
Code comments in this repo are proper sentences, and do not adjoin the comment delimiter (space after `//`).
https://endless-sky.github.io/styleguide/styleguide.xml#Comments